certs.example/saml2019.crt

```diff
@@ -1,21 +0,0 @@
------BEGIN CERTIFICATE-----
```

Comment:
I believe some of the tests use the 2018 and 2019 certs. I think removing these is the right move, but we'll probs wanna update those tests to use the 2020 and 2021 certs